js/classes/entities/midground/weapon/projectile.js

```diff
@@ -86,8 +86,10 @@ class Projectile {
                 SIZE: new Vector(4, 4)
             },
             [Projectile.STONE]: {
-                ACTION: () => {
-                    console.log("stone");
+                ACTION: (target) => {
```

Comment:
I like this! Very cool! Love how you can just change the Action for each ammo. @tclaar